cristin-person/src/main/java/no/unit/nva/cristin/person/institution/fetch/FetchPersonInstitutionInfoHandler.java

```diff
@@ -0,0 +1,98 @@
+package no.unit.nva.cristin.person.institution.fetch;
+
+import static no.unit.nva.cristin.common.ErrorMessages.ERROR_MESSAGE_INVALID_QUERY_PARAMS_ON_PERSON_LOOKUP;
+import static no.unit.nva.cristin.model.Constants.DEFAULT_RESPONSE_MEDIA_TYPES;
+import static nva.commons.core.attempt.Try.attempt;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.google.common.net.MediaType;
+import java.net.HttpURLConnection;
+import java.util.List;
+import no.unit.nva.cristin.common.Utils;
+import no.unit.nva.cristin.person.model.nva.PersonInstitutionInfo;
+import nva.commons.apigateway.ApiGatewayHandler;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.apigateway.exceptions.BadRequestException;
+import nva.commons.core.Environment;
+import nva.commons.core.JacocoGenerated;
+
+public class FetchPersonInstitutionInfoHandler extends ApiGatewayHandler<Void, PersonInstitutionInfo> {
+
+    public static final String PERSON_ID = "person-id";
+    public static final String ORG_ID = "org-id";
+    public static final String PUNCTUATION = ".";
+    public static final String INVALID_ORGANIZATION_ID = "Invalid path parameter for organization id";
+    public static final String INVALID_PERSON_ID = "Invalid path parameter for person id";
+
+    private final transient FetchPersonInstitutionInfoClient apiClient;
+
+    @JacocoGenerated
+    public FetchPersonInstitutionInfoHandler() {
+        this(new Environment());
+    }
+
+    @JacocoGenerated
+    public FetchPersonInstitutionInfoHandler(Environment environment) {
```

Comment:
This constructor is used only by the constructor above

Reply:
I think we needed constructors like these for the template to call the handler?